compiledb/__init__.py

```diff
@@ -37,16 +41,24 @@ def generate_json_compdb(instream=None, proj_dir=os.getcwd(), verbose=False, exc
 
 def write_json_compdb(compdb, compdb_path='compile_commands.json', verbose=False,
                       force=False, pretty_output=True):
-    print("## Writing compilation database with {} entries to {}".format(
-        len(compdb), compdb_path))
+    if not __is_stdout(compdb_path):
+        print("## Writing compilation database with {} entries to {}".format(
+            len(compdb), compdb_path))
 
-    with open(compdb_path, 'w') as outstream:
-        json.dump(compdb, outstream, indent=pretty_output)
-        outstream.write(os.linesep)
+        with open(compdb_path, 'w') as outstream:
+            json.dump(compdb, outstream, indent=pretty_output)
+            outstream.write(os.linesep)
+    else:
+        print("## Writing compilation database with {} entries to <stdout>".format(
+            len(compdb)))
+        print(json.dumps(compdb, indent=pretty_output))
 
 
 def load_json_compdb(compdb_path='compile_commands.json', verbose=False):
     try:
+        if __is_stdout(compdb_path):
```

Comment:
Don't try to read stdout

Reply:
compiledb supports stdin input, wouldn't make sense to try to load from it?